src/PlusDataCollection/WinProbe/vtkPlusWinProbeVideoSource.h

```diff
@@ -195,6 +195,8 @@ class vtkPlusDataCollectionExport vtkPlusWinProbeVideoSource : public vtkPlusUsD
     CFD // Color-Flow Doppler
   };
 
+  PlusStatus SetExtraSourceMode(Mode mode);
```

Comment:
do we plan on doing something with the returned plus status in the future? the functions which call this are `void`